### ISSUE TITLE
FR: make it possible to customize the transitions between pages.

### ISSUE THREAD

**Issue by Ingenu1ty** [saltcorn/saltcorn#1909]
...and between view state changes too 😉

As an example, consider something like this: https://codesandbox.io/s/advanced-react-page-transition-demo-z8hmd 

it's open source and **mit licensed** 
(https://github.com/Steveeeie/react-page-transition) so it would be great to make it possible in the Saltcorn 

p.s. Thank you for your efforts! Saltcorn is awesome!🤩

**Comment by serge402** [comment]
@Ingenu1ty I agree, transition would make Saltcorn and Mobile Apps look and feel way more modern.
